apps/api-wallet/.env

```diff
@@ -3,8 +3,9 @@ DATABASE_NAME=walletmini
 DATABASE_PASSWORD=a2j?XU4^dU?6DmN@
 DATABASE_ROOT=devusr
 DATABASE_PORT=3306
-JWT_SECRET=ichad123
 
+JWT_SECRET=ichad123
```

Comment:
this file should not show here since we added it na sa gitignore

Reply:
done